### DISCUSSION TITLE
Is it possible to setup watchtower to automaticaly update apps?

### DISCUSSION THREAD

**Discussion by arthuragone** [caprover/caprover#1975] [Q&A] [answered]
I use Caprover for 6 months now, and manually check for my app versions from time to time. 
watchtower has been mentioned numerous times on Reddit and I wonder if it is possible to use it with Caprover to update apps installed via Caprover?

**Comment by githubsaturn** [comment, marked as answer]
No, you can't use Watchtower for CapRover. Both control the version of the apps deployed.
